### DISCUSSION TITLE
checking trust tokens

### DISCUSSION THREAD

**Discussion by sergerdn** [abrahamjuliot/creepjs#156] [Ideas]
Hi, there. Firstly I would like to said that I'm not frontend developer and I have a very limited experience with javascript. 
But what about https://web.dev/trust-tokens/? I have check all my Chrome browsers on a few laptops and I have at least one trust token in all Chrome browsers.
![Screenshot 2021-11-18 at 03 07 45](https://user-images.githubusercontent.com/64213648/142302479-c5a30d0b-8297-4c81-bfda-bfffeb89622d.png)
Is there any way to get that token on any website, e.g. https://abrahamjuliot.github.io/creepjs/? If so, if browser haven't any tokens, it is not Chrome browser or we have running Chrome browser in incognito mode or we have running Chrome with very fresh profile or any other cool logic?

Some useful links:
- https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md
- https://www.chromium.org/updates/trust-token
- https://github.com/WICG/trust-token-api

**Comment by abrahamjuliot** [comment]
Thanks for bringing this to my attention. This definitely catches my interest. I plan to dissect this as it becomes more stable.
